flask_mongorest/resources.py

```diff
@@ -332,6 +332,100 @@ def _subresource(self, obj):
         else:
             return None
 
+    def get_field_value(self, obj, field_name, field_instance=None, **kwargs):
```

Comment:
Should add a docstring now :) Including explaining what all the args/kwargs are used for, and what the return value is.

Also add a docstring to the other methods.

If any of the methods below shouldn't be used publicly or shouldn't be subclassable you may want to prefix them with an underscore.